packages/map-template/README.md

```diff
@@ -138,6 +138,7 @@ Note that when using the React component, the properties should conform to JSX p
 |`category`|`category`|`string`|If you want to indicate an active category on the map. The value should be the Key (Administrative ID). |
 |`search-all-venues`|`searchAllVenues`|`bool`|If you want to perform search across all venues in the solution. |
 |`hide-non-matches`|`hideNonMatches`|`bool`|Determine whether the locations on the map should be filtered (only show the matched locations and hide the rest) or highlighted (show all locations and highlight the matched ones with a red dot by default). If set to true, the locations will be filtered. |
+|`show-road-name-labels`|`showRoadNameLabels`|`bool`|A boolean parameter that dictates whether Mapbox road names should be shown. By default, Mapbox road names are hidden when MapsIndoors data is shown. |
```

Comment:
Should write something here about the transition level and the default 17 zoom level.

Reply:
✅